docs/source/stepfunctions.rst

```diff
@@ -0,0 +1,118 @@
+3. Configure jobs configuration file using the following `guide <#jobs-configuration-guide>`_
+4. Deploy a Step Function using :code:`cfncluster stepfunctions`; see `here <commands.html#stepfunctions>`_
+5. Navigate to the Step Function using the deeplink given from the command
+6. Click Start Execution and provide a cluster name via JSON execution input
+
+::
+
+    {
+        "cluster_name": "cfnclusterstepfunctions"
+    }
+
+
+.. image:: https://s3.amazonaws.com/global-cfncluster/doc-images/command_end.gif
+.. image:: https://s3.amazonaws.com/global-cfncluster/doc-images/JSON.gif
+
+Jobs Configuration Guide
+========================
+
+::
+
+    [order]
+    sequential = job1, banana, job2
+
+    [job job1]
+    handler = src/script.sh
+    s3_uri = s3://bucket-to-use/folder/path/to/project
+
+    [job job2]
+    handler = is-this-even-a-job
+    local_path = /path/to/the/job/is-this-even-a-job
+
+    [job banana]
+    handler = long-running-script.sh
+    s3_uri = s3://bucket-to-use/folder/path/to/project
+    wait_time = 240
+
+Sections Options:
+    :code:`[order]` required parameters:
+        * :code:`sequential`: List of job names to schedule sequentially given in the form of a comma separated list; order matters
+
+        ::
+
+            [order]
+            sequential = firstjob, secondjob, thirdjob
+
+        OR
+
+        * :code:`parallel`: List of job names to schedule in parallel given in the form of a comma separated list; order does not matter
+
+        ::
+
+            [order]
+            parallel = paralleljob1, paralleljob2, otherjob
+
+    **IMPORTANT**: either :code:`sequential` or :code:`parallel` must be specified; not both
+    
+    :code:`[job <job_name>]` required parameters:
+        * :code:`s3_uri`: An S3 URI pointing to the script or folder to pacakge for job scheduling and execution
+
+        ::
+
+            [job apple]
+            s3_uri = s3://thebucket/thefolder
+            handler = thescript
+
+        OR
+
+        * :code:`local_path`: A local path (relative to the jobs config file or absolute) pointing to the script or folder for job scheduling and execution
+
+        ::
+        
+            [job banana]
+            local_path = /path/to/the/script
+            handler = script
+
+        AND
+
+        * :code:`handler`: The path and name of the script to run. Since the :code:`s3_uri` and :code:`local_path` can both be directories, this is to specify which file to send off to the scheduler
+
+        ::
+
+            [job carrot]
+            local_path = relative/path/project
+            handler = script/path/in/project.sh
+
+    **IMPORTANT**: either :code:`s3_uri` or :code:`local_path` must be specified; not both
+
+    :code:`[job <job_name>]` optional parameters:
+        * :code:`wait_time`: How long to wait between rechecking the status of the job to see if it's completed; default = 10; range 1-240 due to scheduler limitations
```

Comment:
Period between polling on the status of the job in seconds.

Reply:
much better! also forgot my units 😐